lib/elixir/pages/Operators.md

```diff
@@ -67,12 +69,20 @@ The reason we can compare different data types is pragmatism. Sorting algorithms
 number < atom < reference < function < port < pid < tuple < map < list < bitstring
 ```
 
-When comparing two numbers of different types (a number is either an integer or a float), a conversion to the type with greater precision will always occur, unless the comparison operator used is either `===` or `!==`. A float will be considered more precise than an integer, unless the float is greater/less than +/-9007199254740992.0, at which point all the significant figures of the float are to the left of the decimal point. This behavior exists so that the comparison of large numbers remains transitive.
+When comparing two numbers of different types (a number being either an integer
+or a float), a conversion to the type with greater precision will always occur,
+unless the comparison operator used is either [`===`](`===/2`) or [`!==`](`!===/2`).
+A float will be considered more precise than an integer, unless the float is
+greater/less than `+/-9007199254740992.0` respectively, at which point all the
```

Comment:
backticks added to `+/-9007199254740992.0`
"respectively" added for clarity

Reply:
Why the backticks?

Reply:
True. They shouldn't have been used in this case.